packages/react-composites/src/composites/CallComposite/styles/CallConfiguration.styles.ts

```diff
@@ -125,3 +127,25 @@ export const startCallButtonStyleMobile = mergeStyles({
   width: '100%',
   maxWidth: 'unset'
 });
+
+/* @conditional-compile-remove(video-background-effects) */
```

Comment:
We don't have to CC it if that is not exported publicly, and it is encouraged to reduce CC comments if possible

Reply:
done